frontend/src/container/GridGraphLayout/Graph/index.tsx

```diff
@@ -287,7 +288,10 @@ function GridCardGraph({
 		);
 	}
 
-	if (prevChartDataSetRef?.labels === undefined && queryResponse.isLoading) {
+	if (
+		prevChartDataSetRef?.labels === undefined &&
```

Comment:
can you confirm do we need `prevChartDataSetRef` check also ?

Reply:
Got it, we don't need this one as it is handled down below.

Reply:
I will remove it.